third_party/3/pkg_resources/__init__.pyi

```diff
@@ -202,7 +202,19 @@ class IMetadataProvider:
 
 class ResolutionError(Exception): ...
 class DistributionNotFound(ResolutionError): ...
-class VersionConflict(ResolutionError): ...
+
+class VersionConflict(ResolutionError):
+    @property
+    def dist(self) -> List[Any]: ...
+    @property
+    def req(self) -> List[Any]: ...
```

Comment:
Are those really lists? You could have a look at where `VersionConflict` is constructed to find out the real types of those fields, or just use `Any` for now.

Reply:
Yikes, fixed.